<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MariaDBTestClass</class>
 <widget class="QWidget" name="MariaDBTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MariaDBTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="l_host">
       <property name="text">
        <string>Хост:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lE_host">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_port">
       <property name="text">
        <string>Порт:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lE_port">
       <property name="text">
        <string>3306</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="l_name">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lE_name">
       <property name="text">
        <string>testUser</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="l_password">
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lE_password">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="l_DB">
       <property name="text">
        <string>База данных:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lE_DB">
       <property name="text">
        <string>books</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pB_connect">
       <property name="text">
        <string>Соединиться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_disconnect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Разъединиться</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mariadbtest.qrc"/>
 </resources>
 <connections/>
</ui>
